src/component/app.jsx

```diff
@@ -157,27 +250,56 @@ export default class App extends Component {
                                 <FooterSection type="middle">
                                     <FooterDropDownSection title="Menu">
                                         <FooterLinkList>
-                                            {createListItem('/features', 'Feature Toggles')}
-                                            {createListItem('/strategies', 'Strategies')}
-                                            {createListItem('/history', 'Event History')}
-                                            {createListItem('/archive', 'Archived Toggles')}
-                                            {createListItem('/applications', 'Applications')}
+                                            {createListItem(
```

Comment:
I agree, but this component could be split up into smaller ones, which would decrease the indentation, which would keep it on a single line